packages/syft/src/syft/service/code/user_code.py

```diff
@@ -643,12 +644,15 @@ def add_custom_status(context: TransformContext) -> TransformContext:
             node_id=context.node.id,
             verify_key=context.node.signing_key.verify_key,
         )
-        if node_view in input_keys or len(input_keys) == 0:
```

Comment:
We need to think about how we handle this. Not sure if it would be a security issue to not check this at all

Reply:
@rasswanth-s can you verify this once.

Reply:
I think for now, we could proceed with the change.

One thing which is unclear, to me is the UX 
on how a data owner would identify , if this particular request is a request for some object on my machine, or it is request for data belonging to other people.

That is something, we could discuss.

Reply:
They wouldnt, as they wouldnt even be aware of the request in our use case